tests/unit/test_paginate.py

```diff
@@ -613,12 +613,18 @@ def test_resume_with_multiple_input_keys(self):
         complete = pages.build_full_result()
         self.assertEqual(complete,
                          {"Users": ['User4'],
-                          "Groups": [],
```

Comment:
In this specific test case, yes. But a robust paginator will need to deal with possible absence of keys in any page anyway, and our paginator.build_full_result() will handle it correctly, so it should be fine.